contracts/account-history-contract/src/action/query/get_estaking_rewards.rs

```diff
@@ -17,12 +17,20 @@ pub fn get_estaking_rewards(
 
     let response = querier.get_estaking_rewards(address).unwrap_or_default();
 
-    let fiat_coins = response
-        .to_dec_coin_values(&querier, &generator.metadata.usdc_denom)
-        .unwrap_or_default();
+    let fiat_coins = response.to_dec_coin_values(&querier, &generator.metadata.usdc_denom);
+
+    match fiat_coins {
+        Err(e) => {
+            return Err(StdError::generic_err(format!(
+                "Failed to convert to DecCoinValue {}",
+                e
+            )))
+        }
+        _ => {}
+    }
```

Comment:
```suggestion
    let fiat_coins = response.to_dec_coin_values(&querier, &generator.metadata.usdc_denom).map_err(|e| 
 StdError::generic_err(format!(
                "Failed to convert to DecCoinValue {}",
                e
            )))?;
```
Map error will transform error of to_dec_coin_values to StdError